### PR TITLE
Improve geolocation map markers' labels

### PR DESCRIPTION
**Pull request by lqd**
It now includes the "content", the actual street/etc information, in addition to its owner and title.

Works similarly for Google Maps and OSM markers.

image

image

### CODE REVIEW

**Comment by lqd** [comment]
CI will pass once the fixes in the `v2.4-rc2` branch land on the `main` branch.

File: creme/geolocation/static/geolocation/js/geolocation.js [hunk 1]
```diff
@@ -131,9 +131,22 @@ creme.geolocation.Location = creme.component.Component.sub({
     },
```
Comment (genglert) [inline creme/geolocation/static/geolocation/js/geolocation.js:131]: Update the date in the header.

**Review by genglert — COMMENTED** [review]
- Rebase your PR on branch 'v2.4-rc2'.

**Comment by lqd** [comment]
cc @hsmett

**Review by hsmett — APPROVED** [review]
:tada:

**Review by joehybird — APPROVED** [review]

**Pull request merged**